docs/configuration.rst

```diff
@@ -25,3 +25,11 @@ for more information on how each engine can be configured.
 
 If your job callback is performing a database involved operation, you should check if the CakePHP database connection is still alive.
 It can happen that your database connection timed out, when no jobs were acknowledged for a long time.
+
+.. code:: php
+
+    // Disconnect the connection fist...
+    ConnectionManager::get('default')->disconnect();
+
+    // ...and renew it inside a Table class
+    $this->connection(ConnectionManager::get('default'));
```

Comment:
Any reason why you didn't just show an example using the event system like you did in the comment on the other issue?